lib/display/display.ts

```diff
@@ -31,63 +33,45 @@ function findDependencyLines(
   depGraph: DepGraph,
   options: Options | undefined,
   depsFilePaths?: DepsFilePaths,
-) {
+  fileSignaturesDetails?: FileSignaturesDetails,
+): string[] {
   const showDepsFilePaths = (options && options['print-dep-paths']) || false;
-  const isAllowedToShowDependenciesWithFilePaths =
-    showDepsFilePaths && depsFilePaths && Object.keys(depsFilePaths).length > 0;
+  const showOnlyDeps = (options && options['print-deps']) || false;
 
-  if (isAllowedToShowDependenciesWithFilePaths) {
-    return displayDependenciesWithFilePaths(depGraph, depsFilePaths);
+  if (showDepsFilePaths) {
+    return displayDependencies(depGraph, fileSignaturesDetails, depsFilePaths);
+  } else if (showOnlyDeps) {
```

Comment:
showOnlyDeps feels off, this makes me think that it's not going to show confidence only "dependencies"

whilst "allowing something"... means that we permit also that without exclude something else